apps/agreement/contracts/Agreement.sol

```diff
@@ -1127,9 +1127,7 @@ contract Agreement is IAgreement, AragonApp {
             return false;
         }
 
-        // TODO: update with new ACL version: no need to pass challenger address by parameter
-        bytes memory params = ConversionHelpers.dangerouslyCastUintArrayToBytes(arr(_challenger));
-        return currentKernel.hasPermission(_challenger, address(_disputable), CHALLENGE_ROLE, params);
+        return currentKernel.hasPermission(_challenger, address(_disputable), CHALLENGE_ROLE, new bytes(0));
```

Comment:
To double check, the reason we don't need to add the `challenger` any more is because this address is now available by default in the ACLOracle?

Reply:
Exactly, we are testing that here https://github.com/aragon/aragon-apps/blob/master/apps/agreement/test/agreement/agreement_permissions.js#L153-L206